### ISSUE TITLE
Issues with batch job logs

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-processes-dask#282]
Note: I'm not sure if I should report this issue under this repo, feel free to move it appropriately.

I had a failed batch job on openeo.eodc.eu and noticed the batch job logs look like this (Jupyter notebook on the left, web editor on the right):
![image](https://github.com/user-attachments/assets/b1098a21-14be-4314-9f4e-a3422493fe89)

a couple of issues:
- all log items have level INFO, while there are clearly items that should have level ERROR 
- There is a stack trace, but apparently each line of the stack trace is a separate log item/message. I think that is a bad format to provide the stack trace to the user, as you lose the inherent line oriented structure and context of stack traces. E.g., now you have log messages with just `^^^^^`, which makes no sense. You also lose the indentation, making it hard to parse visually.
- In some messages there is a timestamp, e.g. `2024-09-24 13:03:25,976 - openeo_execut...`. Note that there is a dedicated "time" field for openeo batch job log items

**Comment by m-mohr** [comment]
For the average user the stack traces will just be confusing or at least unhelpful, that's why it's "not recommended". There's not really an alternative right now. If you think they are useful for your users, you can still put them in the message, it's just a recommendation. We never defined an API field for stack traces yet (although there's an issue for it).

**Comment by ValentinaHutter** [comment]
Thanks for pointing this out, I will move this issue to our internal issue tracking system. Feel free to reopen this issue, in case anything related comes up.
